html/partials/users.html.haml

```diff
@@ -0,0 +1,30 @@
+- if users
+  %div
+    %a.scope_anchor{:id => "users"}
+    #users_container.scope
+      = render_partial "scope_header",  { |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.